deploy/kubernetes/overlays/examples_perf/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+bases:
+- ../../base
+resources:
+- configmap.yaml
```

Comment:
i'd prefer just keeping this all in the examples folder.  Users can comment out the kustomizations they don't want there.

Reply:
I tried to add all files to examples folder but then kustomize returns following error:
```
$ kustomize build deploy/kubernetes/overlays/examples | kubectl apply -f -
Error: conflict between map[string]interface {}{"apiVersion":"apps/v1", "kind":"DaemonSet", "metadata":map[string]interface {}{"name":"cadvisor", "namespace":"cadvisor"}, "spec":map[string]interface {}{"template":map[string]interface {}{"spec":map[string]interface {}{"containers":[]interface {}{map[string]interface {}{"args":[]interface {}{"--housekeeping_interval=10s", "--max_housekeeping_interval=15s", "--event_storage_event_limit=default=0", "--event_storage_age_limit=default=0", "--disable_metrics=percpu,hugetlb,sched,tcp,udp,advtcp", "--docker_only", "--store_container_labels=false", "--whitelisted_container_labels=io.kubernetes.container.name, io.kubernetes.pod.name,io.kubernetes.pod.namespace"}, "env":[]interface {}{map[string]interface {}{"name":"LD_LIBRARY_PATH", "value":"/bin/nvidia/lib64/"}}, "name":"cadvisor", "securityContext":map[string]interface {}{"privileged":true}, "volumeMounts":[]interface {}{map[string]interface {}{"mountPath":"/dev", "name":"dev"}, map[string]interface {}{"mountPath":"/bin/nvidia/lib64/", "name":"libnvidia"}}}}, "volumes":[]interface {}{map[string]interface {}{"hostPath":map[string]interface {}{"path":"/dev"}, "name":"dev"}, map[string]interface {}{"hostPath":map[string]interface {}{"path":"/home/kubernetes/bin/nvidia/lib64/"}, "name":"libnvidia"}}}}}} and map[string]interface {}{"apiVersion":"apps/v1", "kind":"DaemonSet", "metadata":map[string]interface {}{"name":"cadvisor", "namespace":"cadvisor"}, "spec":map[string]interface {}{"template":map[string]interface {}{"spec":map[string]interface {}{"containers":[]interface {}{map[string]interface {}{"args":[]interface {}{"--perf_events_config=/etc/config/perf-non-hardware.json"}, "name":"cadvisor", "securityContext":map[string]interface {}{"privileged":true}, "volumeMounts":[]interface {}{map[string]interface {}{"mountPath":"/etc/config/", "name":"perf-volume"}}}}, "volumes":[]interface {}{map[string]interface {}{"configMap":map[string]interface {}{"name":"perf-config"}, "name":"perf-volume"}}}}}}
error: no objects passed to apply
```

It seems that kustomize has a problem with two patches which modify the same fields in different way, e.g. args

Reply:
ah, ok.  In that case, we can have a separate folder